src/components/Timeline/index.jsx

```diff
@@ -13,7 +13,8 @@ import Hashtags from "../Hashtags/index.jsx";
 export default function Timeline() {
   const statusMessages = {
     loading: "Loading",
-    emptArray: "There are no posts yet",
```

Comment:
We are not using this anymore, as we have to new messages to show about posts status

Reply:
ok